source/navier_stokes/parabolic_solver.h

```diff
@@ -37,8 +37,8 @@ namespace ryujin
     class DiagonalMatrix;
 
     /**
-     * Minimum entropy guaranteeing second-order time stepping for the
-     * parabolic limiting equation @cite ryujin-2021-2, Eq. 3.3:
+     * Decond-order time stepping for the parabolic limiting equation @cite
```

Comment:
Second*

Reply:
Good point. I have updated the documentation